src/helm/config/model_metadata.yaml

```diff
@@ -100,6 +100,25 @@ models:
   # - j2-large -> j2-light
 
 
+  # AI Singapore
+  - name: aisingapore/sea-lion-7b
+    display_name: SEA-LION (7B)
+    description: SEA-LION is a collection of language models which has been pretrained and instruct-tuned on languages from the Southeast Asia region. It utilizes the MPT architecture and a custom SEABPETokenizer for tokenization.
+    creator_organization_name: AI Singapore
+    access: open
+    num_parameters: 7000000000
+    release_date: 2023-02-24
```

Comment:
Are these release dates accurate? I haven't been able to find a good source for these... the date can be the date on which the model weights were publicly released (preferred) or the date of the public announcement.